packages/activity/src/index.ts

```diff
@@ -15,7 +15,13 @@
  *
  * ### Cancellation
  *
- * Activities may be cancelled only if they {@link Context.heartbeat | emit heartbeats}.
+ * Activity Cancellation serves three purposes:
+ *
+ * - It lets an Activity know it doesn't need to keep doing work.
+ * - It gives the Activity time to clean up any resources it has created.
+ * - If the Activity accepts Cancellation, the Server will know to retry the Activity.
+ *
+ * Activities may be Cancelled only if they {@link Context.heartbeat | emit heartbeats}.
```

Comment:
You might want to note that this doesn't apply to local activities.

Reply:
Can LAs call `Context.current().heartbeat()`? 

Can LAs receive Cancellation?

Reply:
> Can LAs call `Context.current().heartbeat()`?

No (I think we ignore that for now but we might want to throw there, that's what Python does).

> Can LAs receive Cancellation?

Only in core based SDKs.